### DISCUSSION TITLE
Usability of CSS Battle

### DISCUSSION THREAD

**Discussion by axelf** [cssbattle/forum#364] [General]
Hi,

Either I don't understand the site or the usability is so bad. I can't find my way around. Maybe as a normal member you are so limited that you can do almost nothing. I can't even see my last battles.

**Comment by ngdangtu-vn** [comment]
The site UI is broken or UX fails you, which one do you mean?

**Comment by chinchang** [comment]
Hi @axelf Thanks for pointing out. I actually agree we could do better in terms of seeing your "past play" in the game....your point noted! Just to be sure we are got it right, you wanted to see what all levels/battles you have played right?

**Reply by axelf** [reply]
Yes that's right. Thank you for the info.
